Library/Formula/docker-machine-nfs.rb

```diff
@@ -0,0 +1,16 @@
+class DockerMachineNfs < Formula
+  desc "Activates NFS on docker-machine"
+  homepage "https://github.com/adlogix/docker-machine-nfs"
+  url "https://raw.githubusercontent.com/adlogix/docker-machine-nfs/0.1.0/docker-machine-nfs.sh"
```

Comment:
It’s probably better to use the tarball here so it (automatically) installs the README as well.